addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -58,7 +58,7 @@
 
     <t t-name="portal.Chatter.Attachments">
         <div t-if="attachments.length" class="row">
-            <div t-foreach="attachments" t-as="attachment" class="col-lg-2 col-md-3 col-sm-6">
+            <div t-foreach="attachments" t-as="attachment" class="col-lg-6 col-md-3 col-sm-6">
```

Comment:
I would maybe use the class `col-12 col-sm-6` to have a layout of 1 column on small display and a layout of 2 columns on larger display. With the current classes, we can have a layout of 4 columns on medium size displays (see: `col-md-3`). I feel like there will not be enough space to properly show the attachments given that the modal will roughly have the same dimensions on medium and large displays.

Reply:
Hello @julien-banken 
Changes are done can you please have a look?
Thanks :blush:

Reply:
After given it another shot: I noticed that the template we are changing is also used on the "Reviews" tab of the course. On that view, there is maybe too much space between the elements now 😕

We can maybe rather set the class `col-12 col-sm-6 col-md-4` and add the class `modal-lg` on the media dialog (i.e: the div `.modal-dialog` in `addons/portal_rating/static/src/xml/portal_rating_composer.xml`) to get a larger modal?

That way, the modal will have a width that is similar to the "Reviews" tab and we will be able to properly display 3 columns. The display will then be okay on the modal and the "Reviews" tab :smiley:

Reply:
Side note:

When I:
1. Write a new review
2. Attach some files
3. Send the review
4. Click on the "Modify your review" link
5. Click on an attachment

I get a 404 error. But if I click on the attachment on the "Reviews" tab, I can download the file. I will report the issue :smiley:

Reply:
Hello @julien-banken 

Changes are done.
Can you please have a look?

Thanks :blush: